src/cmd/linuxkit/run_qemu.go

```diff
@@ -147,6 +160,7 @@ func runQemu(args []string) {
 	// options. So this must remain after the `flags.Parse` above.
 	envOverrideBool("LINUXKIT_QEMU_KVM", enableKVM)
 	envOverrideBool("LINUXKIT_QEMU_CONTAINERIZED", qemuContainerized)
+	envOverrideString("LINUXKIT_QEMU_OPTIONS", qemuDirectOptions)
```

Comment:
The original two were added to allow for quick/dirty performance testing of this stuff when linuxkit is used (internally) by the moby tool (we were seeing slowness on CI, where KVM is not an option).

What is the rationale for allowing arbitrary command line overrides via that mechanism?